app/lua/lflash.c

```diff
@@ -257,60 +256,20 @@ LUALIB_API int lua_lfsreload (lua_State *L) {
   NODE_ERR(msg);
 
   while (1) {}  // Force WDT as the ROM software_reset() doesn't seem to work
-  return 0;
 }
 
 
-/*
- * If the arg is a valid LFS module name then return the LClosure
- * pointing to it. Otherwise return:
- *  -  The Unix time that the LFS was built
- *  -  The base address and length of the LFS
- *  -  An array of the module names in the LFS
- */
-LUAI_FUNC int lua_lfsindex (lua_State *L) {
-  int n = lua_gettop(L);
+LUA_API void lua_getlfsconfig (lua_State *L, int *config) {
+  if (!config)
+    return;
+  config[0] = (int) flashAddr;                   /* LFS region mapped address */
+  config[1] = flashAddrPhys;                 /* LFS region base flash address */
+  config[2] = G(L)->LFSsize;                        /* LFS region actual size */
+  config[3] = cast(FlashHeader *, flashAddr)->flash_size;  /* LFS region used */
```

Comment:
Shouldn't there be a check if LFS is flashed at all (as in the Lua53 Version)?

Reply:
replaced by:
```C
  config[3] = (G(L)->ROstrt.hash) ? cast(FlashHeader *, flashAddr)->flash_size : 0;  
```
The used is 0 if the ROstrt is NULL.